bindings/nodejs/lib/types/client/bridge/index.ts

```diff
@@ -3,7 +3,7 @@ import type {
     __GetBasicOutputIdsMethod__,
     __GetOutputMethod__,
     __GetOutputsMethod__,
-    __GenerateAddressesMethod__,
+    __GenerateEd25519AddressesMethod__,
```

Comment:
Doesn't EVM need to be added here?

Reply:
No, this is the client and I only added it to the account for old bindings.

Reply:
It's a bit confusing tbh, I'd expect them to always be together, not sure why we would either not add it, or keep ed25519

Reply:
Remove ed25519 https://github.com/iotaledger/iota-sdk/issues/492 :trollface: